client/src/components/ProgressTrackingButtons.js

```diff
@@ -1,23 +1,47 @@
 import React from "react";
 import TrackBtn from "./TrackBtn";
 import "./BtnContainer.css";
-export default function ProgressTrackingButtons({ ability, updateAbility }) {
+export default function ProgressTrackingButtons({
+  ability,
+  updateAbility,
+  deselect,
+  student_id,
+  learningObjId,
+}) {
   return (
     <div className="btn-container">
       <TrackBtn
-        classNameBtn={ability === 0 ? "btn-red" : ""}
-        onClick={() => updateAbility(0)}
+        classNameBtn={ability === 0 ? "btn-red" : "btn-white"}
+        onClick={() =>
+          ability == null
+            ? updateAbility(0)
+            : ability == 0
+            ? deselect(student_id, null, learningObjId)
+            : updateAbility(0)
+        }
```

Comment:
This (and the other below) ternaries are very confusing to me. I'm not sure you need a different api endpoint and specific `deselect()` function (can't you just use `updateAbility(null)`?) but even if you do have some reason to use a new `deselect()` function, can't you do
```suggestion
        onClick={() =>
          ability === 0
            ? deselect(student_id, null, learningObjId)
            : updateAbility(0)
        }
```